packages/client/hmi-client/src/page/project/components/tera-upload-resources-modal.vue

```diff
@@ -153,6 +154,11 @@ async function upload() {
 				const newAsset = useProjects().addAsset(assetType, id);
 				if (name && name.toLowerCase().endsWith('.pdf')) {
 					extractPDF(id);
+				} else if (
+					(name && name.toLowerCase().endsWith('.txt')) ||
+					(name && name.toLowerCase().endsWith('.md'))
+				) {
+					modelCard(id);
```

Comment:
Why this is done here?

Reply:
For txt and md we dont need to wait for extractions since the text field is already available

Reply:
I guess for these files in particular, this could be done in the backend

Reply:
That would be great indeed.